README.md

```diff
@@ -96,19 +40,17 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Animals Table]**
+- **[Queries]**
+- **[Screenshopts of Queries]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Since this application is not deployed, kindly remove the **Live Demo** section to avoid having a broken link in your README file.